assets/js/contact_us_form_element.ts

```diff
@@ -81,6 +94,14 @@ class ContactUsFormElement extends HTMLElement {
     this.maintenanceAlert.removeAttribute('hidden');
   }
 
+  showOutageAlert() {
+    if (!this.unplannedOutageAlert) {
+      return;
+    }
+
+    this.unplannedOutageAlert.removeAttribute('hidden');
+  }
+
```

Comment:
Based on the view template, the alert would always be present and visible in an unplanned outage, so we don't need to explicitly show the alert, do we?

```suggestion
```

Reply:
No, I had not about this route. The less javascript, the better